### PR TITLE
Update v1.6.0

### PR DESCRIPTION
**Pull request by nauttiilus**
- Feature: Proof-of-GPU benchmark implementation
- Feature: Improved incentive mechanism - GPU specs based scoring
- Fixes to the register API - dashboard interaction
- Other (logging, opencompute dashboard, readme, etc.)

### CODE REVIEW

File: neurons/Validator/database/pog.py
Comment (mo0haned) [inline neurons/Validator/database/pog.py:1]: bittensor and DateTime are not used

File: neurons/Validator/pog.py
Comment (mo0haned) [inline neurons/Validator/pog.py:1]: paramiko, os , time, blake3 and torch  are not used

File: neurons/Validator/database/pog.py
Comment (nauttiilus) [reply neurons/Validator/database/pog.py:1]: Thanks, will be cleaned up in the next update.

File: neurons/Validator/pog.py
Comment (nauttiilus) [reply neurons/Validator/pog.py:1]: Thanks, will be cleaned up in the next update.

**Review by bigsat309 — APPROVED** [review]

File: neurons/Miner/container.py
Comment (mo0haned) [inline neurons/Miner/container.py:1]: at line 250 should we at least log that?

**Review by mo0haned — APPROVED** [review]

**Review by mo0haned — APPROVED** [review]

**Pull request merged**